data/_models/shared/Collections.csv

```diff
@@ -0,0 +1,207 @@
+Collections,,study type,ontology,,,CatalogueOntologies,Study types,,,,,Select 1 of the following types of study,"DataCatalogueFlat,EMA",['Study'].includes(collectionType?.name)
+Collections,,study type other,,,,,,,,,,"If other, describe the type of study","DataCatalogueFlat,EMA",['Study'].includes(collectionType?.name)
+Collections,,datasource type,ontology_array,,,CatalogueOntologies,Datasource types,,,,dcterms:type,Which of the following families of databanks best describe this data source,"DataCatalogueFlat,EMA","['Databank', 'Data source'].includes(collectionType?.name)"
+Collections,,datasource type other,text,,,,,,,,,"If other, describe the type of datasource","DataCatalogueFlat,EMA","['Databank', 'Data source'].includes(collectionType?.name)"
+Collections,,keywords,text,,,,,,,,dcat:keyword,Keywords to increase findability of this resource. Try to use words that are not used in the description,DataCatalogueFlat,"['Cohort', 'Databank', 'Data source'].includes(collectionType?.name)"
+Collections,,website,hyperlink,,,,,,,,dcat:landingPage,Link to the website or homepage,"DataCatalogueFlat,EMA",
+Collections,,lead organisation,ref_array,,,,Organisations,,,,dcterms:creator,lead organisation (e.g. research department or group) for this resource,"DataCatalogueFlat,EMA","['Cohort', 'Databank', 'Data source', 'Model', 'Network', 'Study'].includes(collectionType?.name)"
+Collections,,additional organisations,ref_array,,,,Organisations,,,,dcterms:contributor,List the names of any additional organisations that contributed to the resource,"DataCatalogueFlat,EMA","['Cohort', 'Databank', 'Data source', 'Model', 'Network', 'Study'].includes(collectionType?.name)"
+Collections,,other organisations,text,,,,,,,,,List the names of any other organisations that are not listed and contributed to this resource,EMA,['Study'].includes(collectionType?.name)
+Collections,,other organisations,text,,,,,,,,dcterms:contributor,List the names of any other organisations that are not listed and contributed to this resource,EMA,['Network'].includes(collectionType?.name)
+Collections,,description,text,,,,,,,,dcterms:description,Short description,"DataCatalogueFlat,EMA",
+Collections,,data collection description,text,,,,,,,,,Describe the process of collection and recording of data.,"DataCatalogueFlat,EMA","['Databank', 'Data source'].includes(collectionType?.name)"
+Collections,,date established,date,,,,,,,,,"Date when the data source was first established. If the exact day of the month is not known, please enter 15. If the exact month is not known then please enter 15/06","DataCatalogueFlat,EMA","['Databank', 'Data source'].includes(collectionType?.name)"
+Collections,,start data collection,date,,,,,,,,dcat:startDate,"The date when data started to be collected or extracted. If the exact day of the month is not known, please enter 15. If the exact month is not known then please enter 15/06","DataCatalogueFlat,EMA","['Databank', 'Data source'].includes(collectionType?.name)"
```

Comment:
delete here and force at least one collection event? maybe should then be more genericly named?